pkg/plugin/istio/util.go

```diff
@@ -17,11 +17,69 @@ limitations under the License.
 package istio
 
 import (
+	"bytes"
+	"io/fs"
+	"text/template"
+
+	"github.com/Masterminds/sprig/v3"
+
+	"kurator.dev/kurator/manifests"
 	"kurator.dev/kurator/pkg/client"
 	"kurator.dev/kurator/pkg/generic"
 	"kurator.dev/kurator/pkg/util"
 )
 
+const (
+	eastEestTemplateFileName = "profiles/eastwest.tmpl.yaml"
+	exposeServicesFileName   = "profiles/expose-services.yaml"
+)
+
 func waitIngressgatewayReady(client *client.Client, opts *generic.Options, cluster string) error {
 	return util.WaitMemberClusterPodReady(client, cluster, istioSystemNamespace, "app=istio-ingressgateway", opts.WaitInterval, opts.WaitTimeout)
 }
+
+func waitEastwestgatewayReady(client *client.Client, opts *generic.Options, cluster string) error {
+	return util.WaitMemberClusterPodReady(client, cluster, istioSystemNamespace, "app=istio-eastwestgateway", opts.WaitInterval, opts.WaitTimeout)
+}
+
+func exposeServicesFiles() (string, error) {
+	fsys := manifests.BuiltinOrDir("")
+	out, err := fs.ReadFile(fsys, exposeServicesFileName)
+	if err != nil {
+		return "", err
+	}
+
+	return string(out), nil
+}
+
+type meshOptions struct {
```

Comment:
@hzxuzhonghu @prodanlabs  do you have any good idea on how to open this configuration to end-users.
for example, cluster1 and cluster1 on network1, and cluster3 on network2.

Reply:
What about using `clusterName`

Reply:
I can understand 
```
	ClusterName string
	Network     string
```

but not sure why other two fields are necessary

Reply:
before https://github.com/karmada-io/karmada/issues/2236 is fixed, we use `meshConfig` instead of `annotation` on `istio-system` namespaces.
`Networks` is aggregated from  `ClusterName` and `Network`  to render iop file.